translator/translate/otel/receiver/adapter/translators.go

```diff
@@ -71,13 +65,13 @@ var (
 	otelReceivers = collections.NewSet[string](common.OtlpKey, common.JmxKey, common.PrometheusKey)
 )
 
-// FindReceiversInConfig looks in the metrics and logs sections to determine which
-// plugins require adapter translators. Logs is processed first, so any
-// colliding metrics translators will override them. This follows the rule
-// setup.
+// FindReceiversInConfig looks in the metrics section to determine which
+// plugins require adapter translators. Log plugins are not processed here
+// as they have their own dedicated OTEL pipelines.
```

Comment:
nit: `files` and `windows_events` are not OTEL at all, so it's a bit misleading.